packages/contracts-bedrock/src/L1/OPStackManager.sol

```diff
@@ -97,6 +101,15 @@ contract OPStackManager is ISemver {
         address resolvedDelegateProxy;
     }
 
+    /// @notice Inputs required when initializing the OPStackManager. To avoid 'StackTooDeep' errors,
+    /// all necessary inputs (excluding immutables) for initialization are bundled together in this struct.
+    struct InitializerInputs {
```

Comment:
Might want to remove this based on decision made between initialize vs setRelease.

Reply:
Keeping, not using setRelease anymore.
https://github.com/ethereum-optimism/optimism/pull/11875/commits/a77e799e4e37389132993e8883cd14678ff78d23